src/components/requests/drawing.vue

```diff
@@ -9,27 +15,38 @@
       channel.postRequest('DrawTools.StartDrawingRequest', data);"></code-component>
     </div>
     <div id="StopDrawingRequest">
-      <button class="btn btn-primary exampleready" @click="stopDrawing">DrawTools.StopDrawingRequest - cancel current drawing</button>
+      To stop current drawing progress send <code>StopDrawingRequest</code>.
+      <button class="btn btn-primary exampleready" @click="stopDrawing">DrawTools.StopDrawingRequest</button>
       <div>
         <a id="stopDrawing" href="" @click="getLink">To the documentation</a>
       </div>
       <code-component snippet="var data = ['my functionality id'];
       channel.postRequest('DrawTools.StopDrawingRequest', data);"></code-component>
     </div>
     <div id="StopDrawingRequestClear">
+      To clear drawing from the map send <code>StopDrawingRequest</code> with second member of data parameter given to StopDrawingRequest set as true.
       <button class="btn btn-primary exampleready" @click="stopDrawingClear">DrawTools.StopDrawingRequest - clear drawings</button>
       <code-component snippet="var data = ['my functionality id', true];
       channel.postRequest('DrawTools.StopDrawingRequest', data);"></code-component>
     </div>
+    <div id="DrawingEvent">
+      Everytime mouse is moved on the map DrawingEvent occurs while drawing is in progress and contains all the information regarding current drawing. To ensure that logging isn't clogged check if <code>DrawingEvent.finished</code> is set to <code>true</code> and only log when drawing is finished.
+      <div>
+        <a id="drawingEvent" href="" @click="getLink">To the documentation</a>
```

Comment:
The current examples use `<DocumentationLink type="request" apiDoc="mapping/drawtools/request/startdrawingrequest.md">Documentation for {{ requestName }}</DocumentationLink>` instead of adding a click-handler to links. Using the custom tag allows us to change the UI for documentation links by changing the tag-implementation and not using click-handlers fixes an issue where right-/ctrl-clicking the link works as expected in a browser.

Reply:
Fix'd.